crates/iota-indexer/src/apis/extended_api.rs

```diff
@@ -155,6 +155,14 @@ impl ExtendedApiServer for ExtendedApi {
             .await?;
         Ok(latest_checkpoint.network_total_transactions.into())
     }
+
+    async fn get_participation_metrics(&self) -> RpcResult<ParticipationMetrics> {
+        let participation_metrics = self
+            .inner
+            .spawn_blocking(|this| this.get_participation_metrics())
+            .await?;
+        Ok(participation_metrics)
```

Comment:
Nit: I saw that we could shorten a bit the function impl.

```suggestion
        self.inner
            .spawn_blocking(|this| this.get_participation_metrics())
            .await
            .map_err(Into::into)
```